doc/users/navigation_toolbar.rst

```diff
@@ -99,7 +99,8 @@ Close all plots                    **shift** + **w**
 Constrain pan/zoom to x axis       hold **x** when panning/zooming with mouse
 Constrain pan/zoom to y axis       hold **y** when panning/zooming with mouse
 Preserve aspect ratio              hold **CONTROL** when panning/zooming with mouse
-Toggle grid                        **g** when mouse is over an axes
+Toggle major grid                  **g** when mouse is over an axes
+Toggle major and minor grid        **G** when mouse is over an axes
```

Comment:
Here you say "major and minor grids", but in the api_changes, you say that "G" will toggle minor grids.